TestApp/app/components/DataDocumentListView.js

```diff
@@ -33,9 +54,88 @@ export class DataDocumentListView extends Component {
                     }
               keyExtractor={(item, index) => index.toString()}
             />
+            <Modal
+              animationType="slide"
+              transparent={false}
+              visible={this.state.modalVisible}
+              onRequestClose={() => {}}
+            >
+              <View style={SharedStyles.container}>
+                <SectionList
+                  renderItem={({ item }) => <Text style={[SharedStyles.item, SharedStyles.itemTitle]}>{item}</Text>}
+                  keyExtractor={(item, index) => item + index}
+                  sections={[
+                            {
+                                data: [
```

Comment:
nit-picking about the formatting, the indentation of `data` and `renderItem` doesn't look right. It would be nice to fix them for readability.

Reply:
But that was ESLint that formatted it in a way that fixed formatting erorrs

Reply:
Try replacing
```
{
    data: [
        {
        title: 'ID',
        value: this.state.currentDocument == null ? '' : this.state.currentDocument.id,
        },
],
renderItem: renderDocumentItem
},
```
with this:
```
{
    data: [{
        title: 'ID',
        value: this.state.currentDocument == null ? '' : this.state.currentDocument.id,
        }],
    renderItem: renderDocumentItem
},
```

Reply:
Can we fix all occurrences?